auth-web/src/components/auth/AddNameRequestForm.vue

```diff
@@ -170,12 +170,13 @@ export default class AddNameRequestForm extends Vue {
               },
               incorporationApplication: {
                 nameRequest: {
+                  legalType: LegalTypes.BCOMP,
```

Comment:
Maybe ahead of myself here, but future state this will be dynamic? Can Cooperative or one of the other gazillion entity types follow the same NR flow?

Reply:
AFAIK, users will go through the Name Request UI to both specify named/numbered business as well as entity type (aka legal type). And that's why this nameRequest object is expected to be here.
Also, Entities Team will take over at least some of this code, so we can refactor it to whatever makes most sense.

Reply:
Awesome, thanks!